src/inventory/reports.py

```diff
@@ -73,4 +79,106 @@ def execute(self, report_file_name: str) -> str:
 
         _logger.info(f"completed file upload")
 
-        return f"https://{target_bucket}.s3.amazonaws.com/{report_s3_key}"
\ No newline at end of file
+        return f"https://{target_bucket}.s3.amazonaws.com/{report_s3_key}"
+
+
+def send_email_with_attachment(file_path):
+    # Won't be executed if RECIPIENT or SENDER empty
+    if not os.environ.get("RECIPIENT") or not os.environ.get("SENDER"):
+        _logger.info(f"No email source or target. Skip")
+        return 0
+
+    # Amazon SES Documentation
+    # https://docs.aws.amazon.com/ses/latest/dg/send-email-raw.html#send-email-raw-api
+
+    from_email = os.environ.get("SENDER")
+    from_email_friendly_name = os.environ.get(
+        "SENDER_FRIENDLY_NAME", 'SRE Reports')
```

Comment:
Is this right? i haven't found the definition of "SENDER_FRIENDLY_NAME"

Reply:
Yes, it's okay. I left the default `'SRE Reports'` value if `SENDER_FRIENDLY_NAME` wasn't provided. It can be added in ENV VARs of a lambda function